app.ts

```diff
@@ -10,9 +11,8 @@ app.use("/public", express.static(path.join(__dirname,"public")));
 
 app.use(express.json());
 
-app.use(express.urlencoded({ extended: true}));
+app.use(express.urlencoded({ extended: false}));
```

Comment:
why have you changed this to false? Delete this line as extended is set to true on line 29.